storage/dkg.go

```diff
@@ -89,5 +95,5 @@ type EpochRecoveryMyBeaconKey interface {
 	// from Epoch Fallback Mode. State transitions are allowed if and only if the current state is not equal to
 	// [flow.RandomBeaconKeyCommitted]. The resulting state of this method call is [flow.RandomBeaconKeyCommitted].
```

Comment:
```suggestion
	// from Epoch Fallback Mode. The resulting state of this method call is [flow.RandomBeaconKeyCommitted].
	// State transitions are allowed if and only if the current state is not equal to [flow.RandomBeaconKeyCommitted]. 
	// Repeated calls for the same epoch are idempotent, if and only if the provided EpochCommit confirms the already
	// committed key (error otherwise).
```